src/usePerpsGetSettlementStrategy.ts

```diff
@@ -0,0 +1,50 @@
+import { useQuery } from '@tanstack/react-query';
+import { ethers } from 'ethers';
+import { useErrorParser } from './useErrorParser';
+import { useImportContract } from './useImports';
+import { useSynthetix } from './useSynthetix';
+
+export function usePerpsGetSettlementStrategy({
+  provider,
+  settlementStrategyId,
+  market,
```

Comment:
here too, and in other places
```suggestion
  perpsMarketId,
```
Would be good to have a clear separation where we use `perpsMarketId`, and where it is `spotMarrketId` (or would be better to call it `synthId`)

Reply:
done.
Also renamed `marketId` => `synthMarketId` where **Spot**.